freebridge-ai/recommendation.py

```
@@ -0,0 +1,54 @@
+        context = "\n\n".join(doc.page_content for doc in docs)
+        
+        # 4. LLM 호출
+        formatted_prompt = prompt.format(context=context)
+        recommendations = structured_llm.invoke(formatted_prompt)
+
+        return {
+            "success": True,
+            "data": recommendations
+        }
+
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)